test/powershell/Modules/Microsoft.PowerShell.Core/PSSessionConfiguration.Tests.ps1

```diff
@@ -566,6 +579,15 @@ namespace PowershellTestConfigNamespace
                     $Result.Session.UseSharedProcess | Should be $UseSharedProcess
                 }
 
+                It "Validate Set-PSSessionConfiguration -PSVersion" {
```

Comment:
This test is very similar to the test at L504. Can you use `-testcases`?

Reply:
I think it's fine to keep it as is. This `Context` is for `Set-PSSessionConfiguration` and the one above is in the `Context` of `Register-PSSessionConfiguration`.